packages/store/tests/ensure-store.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { TestBed } from '@angular/core/testing';
+import { State, NgxsModule, ensureStoreMetadata, Action } from '@ngxs/store';
+
+describe('Ensure store for plugins', () => {
+  @State({ name: 'count', defaults: 0 })
+  class CountState {
+    @Action({ type: 'increment' })
+    public addOne(): void {}
+
+    @Action({ type: 'increment' })
+    public addTwo(): void {}
+  }
+
+  beforeAll(async () => {
+    await TestBed.configureTestingModule({
```

Comment:
This is not needed. `compileComponents` is only needed if you pass `templateUrl` to the `@Component`.